### DISCUSSION TITLE
How to run Triggerer in docker

### DISCUSSION THREAD

**Discussion by sebukoleth** [apache/airflow#29887] [Q&A]
The aim is to have only Triggerer process running in a docker. Looking at Airflow's docker-compose script the command used is:
`airflow jobs check --job-type TriggererJob`
However it looks like we can also do `airflow triggerer`
Which one is the "right way"? What is the difference - if any - in behaviour between the two?
Lastly, can we expand Airflow documentation with this information?

**Comment by potiuk** [comment]
`airflow jobs check --job-type TriggererJob` - you misunderstood this. This is liveness check.

it is described in our docs https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#jobs well enough.

But if you think things are unclear - you can always add clarifying documentation, in the place that you were looking for it now that you know. It's super easy, Each of our documentation pages contains "Suggest a change on this page" button at bottom-right and when you click it, PR is opened with the doc page that you can add any change.

Airflow is created by > 2300 contributors - most of them people like you who had a problem with code or docs, and after they got helped, they contributed updates to the documentation in the way and place that would help people like them to understand it better. You are likely the best person in the world that would know where you looked and did not find the right docs, and adding it back in return for a free product is the easiest way to give back for the volunteer work of those > 2300 to create airflow.
